infrastructure/lambda/template.yaml

```diff
@@ -759,6 +763,14 @@ Resources:
       Value: !Sub "{{resolve:ssm:/${CommonStackName}/PersonIdentityTableName}}"
       Description: Table name to retrieve addresses from store
 
+  AddressTtlParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: !Sub "/${AWS::StackName}/AddressTtl"
```

Comment:
I guess this is following existing patterns by using an SSM param, but we have SSM rate limiting issues. Do you think it really needs to be one or could it be a constant?

Reply:
Nope, I just followed the existing patterns. I'm happy to move it just to be a constant in the code

Reply:
Oh actually, the TTL for this table should always be the same as the session and person identity tables because it's all the same data - if a user has a record in the address table but not the session table they're not going to be able to do anything. Shall we just reuse the existing `SessionTtl` SSM param? 

In the mid to long term future we will be migrating a lot of these SSM params to app config ([ADR 188](https://github.com/govuk-one-login/architecture/blob/26245cddb0b81326498e3df4a5a75b2633297c8e/adr/0188-application-configuration-feature-flags.md)) so probably makes sense just to follow the patterns laid out for now